src/components/experienceCard/ExperienceCard.css

```diff
@@ -88,12 +88,12 @@
 }
 
 .experience-card {
-  min-width: 350px;
   /* height: 170px; */
+  min-width: 80vw;
   margin-bottom: 20px;
-  border-radius:10px;
+  border-radius: 10px;
```

Comment:
Can you please remove unnecessary changes in this commit caused by space issues?

It is creating lot of confusion in understanding the changes.